### DISCUSSION TITLE
笔记树 修改字体大小不生效

### DISCUSSION THREAD

**Discussion by dbkuaizi** [tamlok/vnote#1950] [Q&A] [answered]
通过修改qss 实现了颜色的变化，但字体大小设置 没有生效，想把笔记树的字体改大一些 应该怎么设置？

![QQ图片20211128125842](https://user-images.githubusercontent.com/44014601/143730223-581e7aaf-0519-45e6-b114-c550571577c5.jpg)

顺便问一下 如果做了一个新主题 有类似于在线主题商店那样的地方 分享出来吗

**Comment by feloxx** [comment, marked as answer]
我觉得可以vnotex中开个新仓库，放主题的，然后vnote里来做个读这个仓库的主题的代码。

**Reply by tamlok** [reply]
文档更新了，看一下设置方法。在线主题后面会做
